hack/tools/README.md

```diff
@@ -0,0 +1,65 @@
+# run.py
+
+A handy tool to run tests in and interact with Equinix devices.
+
+```bash
+./hack/tools/run.py
+Usage: run.py [OPTIONS] COMMAND [ARGS]...
+
+  General thing doer for flintlock
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  create-device
+  delete-device
+  run-e2e
+```
+
+Run `--help` on all those subcommands to see the run options.
+
+For all commands you will need your Organisation ID and to set your API token
+in the environment as `METAL_AUTH_TOKEN`.
+
+## Running the e2es
```

Comment:
Reading this and following on from previous comment it does feel like this should sit under `test/`